db/fixtures/classifications.yml

```diff
@@ -1070,3 +1070,20 @@
   :parent_id: 0
   :default: true
   :single_value: "0"
+- :description: V2V - Transformation Status
```

Comment:
Drop the `V2V` from the naming.  The point of using `Transformation` was to make it generic so it can be reused for other operations.  P2V for example.

The existing "v2v" tags are specifically for v2v operations and will eventually be replaced with backend operations once we can properly support them with RHV inventory.